scipy/spatial/transform/2DTransfrom.py

```diff
@@ -0,0 +1,40 @@
+#2D matrices used for linear transformation in linear algebra 
+
+import numpy as np
+import math 
+
+
+def scaling(scaling_factor):
+	return(scaling_factor*(np.identity(2))) #This returns a scaling matrix
+
+def rotation(angle):
+	arr = np.empty([2, 2])
+
+	arr[0][0] = math.cos(angle)
+	arr[0][1] = -math.sin(angle)
+	arr[1][0] = math.sin(angle)
+	arr[1][1] = math.cos(angle)
+
+	return arr #This returns a rotation matrix
```

Comment:
If this is going to be worth including (anywhere) in scipy, it's probably worth makign it broadcasting:
```python
c = np.cos(angle)
s = np.sin(angle)
arr = np.empty(c.shape + (2, 2), dtype=c.dtype)
arr[..., 0, 0] = c
arr[..., 0, 1] = -s
arr[..., 1, 0] = s
arr[..., 1, 1] = c
```

Reply:
I have made the required changes.